test/python/fe_api/grouped_gemm/test_grouped_gemm_glu_jax.py

```
@@ -218,3 +218,58 @@ def test_grouped_gemm_glu_jax_errors():
+    allocator passes either way -- fresh device memory reads as zeros -- so this one
+    dirties the allocator first and checks only the contract that still holds.
+    """
+    skip_unless_sm100()
+    from cudnn import grouped_gemm_glu_jax_sm100, grouped_gemm_glu_wrapper_sm100
+
+    m, n_full, k, experts = 1024, 256, 128, 2
+    a_np, b_storage_np, _, alpha_np, prob_np = make_problem(m, n_full, k, experts)
+    # 256-aligned, and the last offset stops short of m, so the tail is never addressed
+    offsets_np = np.array([256, 512], dtype=np.int32)
+
+    to_torch = lambda x: torch.from_numpy(x.view(np.uint8)).view(torch.bfloat16).cuda()
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `to_torch` as a `def`

(E731)